src/afl-fuzz.c

```diff
@@ -266,11 +273,14 @@ int main(int argc, char **argv_orig, char **envp) {
   afl->init_seed = tv.tv_sec ^ tv.tv_usec ^ getpid();
 
   while ((opt = getopt(argc, argv,
-                       "+c:i:I:o:f:m:t:T:dnCB:S:M:x:QNUWe:p:s:V:E:L:hRP:")) > 0)
+                       "+c:i:I:o:f:m:t:T:dnCB:S:M:x:QNUWe:p:s:V:E:L:hRP:")) >
+         0) {
```

Comment:
really ugly formatting IMO

Reply:
I agree. but its the formatter so what do?

Reply:
Never look at it again...

Reply:
We could 
- raise the number of allowed chars per line
- declare the ops as variable above the loop
- never look at it again